cli/src/main/kotlin/uk/gov/justice/digital/cli/client/BlockingDomainClient.kt

```diff
@@ -67,8 +70,12 @@ class BlockingDomainClient : DomainClient {
         HttpRequest.newBuilder(uri)
             .header(ACCEPT, APPLICATION_JSON)
             .header(USER_AGENT, "domain-builder-cli/v0.0.1")
+            .withCustomHeader(TraceIdHeader())
```

Comment:
Just a thought here for the traceId value. Would it be helpful to include any more information  in the trace, which might be beneficial for identifying issues apart from only the random UID part?

Reply:
Good question. I think this will be enough to get us started. I'll be keeping an eye on this and if any further info is needed we can add this in as we go. Will resolve this one for now. Also replied on slack so shout there if you have any concerns.